searx/engines/pixiv.py

```diff
@@ -24,6 +25,22 @@
 base_url = "https://www.pixiv.net/ajax/search/illustrations"
 Proxy = Any
 
+# Check for working proxies
+def check_proxies(proxies):
+    working_proxies = []
+
+    for proxy in proxies:
+        proxy_test_url = f"{proxy}/c/250x250_80_a2/img-master/img/2024/02/12/17/41/49/115989389_p0_square1200.jpg"
+
+        try:
+            test_response = httpx.get(proxy_test_url)
```

Comment:
You should use `get` from `searx.network`.

Reply:
When you use the get method from SearXNG's network stack, don't import the `get` / import the complete module and use it like shown in the example below:

```python
from searx import network
network.get(proxy_test_url)
```

Reply:
> tried your example but got an error:

yes, its about the horrible monkey patching we have .. leave it as is now, I will fix it in my review .. BTW: thanks a lot for implementing the `init()` function :)